src/screens/Asset/Asset.js

```diff
@@ -409,9 +418,18 @@ class AssetScreen extends React.Component<Props, State> {
               isReceiveDisabled={!isReceiveActive}
               showButtons={isSynthetic ? ['receive'] : undefined}
             />
-            {!isSendActive &&
-              <SWActivationCard />
+            {isKeyBasedActive &&
+              <TransferButtonWrapper>
+                <Button
+                  title="Transfer to Smart Wallet"
+                  onPress={() => this.goToSendTokenFlow(assetData)}
```

Comment:
Hmmm, why not to go straight to confirmation screen with SW selected as a receiver and value set to all (if we are willing to encourage users to transfer all related asset value?
Or at least to amount selection screen with SW already selected as a receiver (jumping one step further so no extra click would be needed)? :)

Reply:
Just read the last comment in linear (must have missed that before) :))
Still weird we are not doing this click for a user, tho :)))